### DISCUSSION TITLE
How to add delay to script

### DISCUSSION THREAD

**Discussion by ulle55** [arendst/Tasmota#20704] [Q&A]
Hi. I'm new to Tasmota, so this is properly an easy thing.
I'm using Tasmota to read a Kamstrup meter, and that works fine.
But it's reading all the time, and I would like to have a delay in the script, so it only reads every 5 minutes or less, just so I don't drain the battery in the meter.

This is the script code I use.

```
>D
>B
=>sensor53 r
>M 1
+1,3,kN2,0,1200,KSMC403,1,5,3F1004003C005600570059,3F1003004A0044007B
1,3F10003Ckstr@i0:1000,Wärmemenge,MWh,HeatEnergyE1,3
1,3F10x08xx0056kstr@i0:100,Vorlauftemp.,°C,PreTemp,2
1,3F10x15xx0057kstr@i0:100,Rücklauftemp.,°C,AftTemp,2
1,3F10x22xx0059kstr@i0:100,Temp.diff.,°C,DifTemp,2
1,3F10004Akstr@i1:1,Fließgeschw.,l/h,Flow,0
1,3F10x06xx0044kstr@i1:100,Volumenstrom,m³,Volume,2
1,3F10x15xx007Bkstr@#,max. Fluss am,,MaxFlowDate,0
#
```

**Comment by gemu2015** [comment]
this parameter defines update interval, 5 means every 500 ms
+1,3,kN2,0,1200,KSMC403,1,**5**,3F1004003C005600570059,3F1003004A0044007B
however this parameter can only be 255, so 25.5 seconds is maximum interval here.

if you want longer intervals you need some code in script.  (see examples in sml docs)

**Reply by Brinkmann1** [reply]
I was not able to find an example in the sml docs can you please provide one. Thank you. BR Ole
